src/zenml/zen_server/deploy/helm/templates/server-db-job.yaml

```diff
@@ -61,68 +52,21 @@ spec:
-            {{- end }}
-            {{- if .Values.zenml.auth.externalUserInfoURL }}
-            - name: ZENML_SERVER_EXTERNAL_USER_INFO_URL
-              value: {{ .Values.zenml.auth.externalUserInfoURL | quote }}
-            {{- end }}
-            {{- if .Values.zenml.auth.externalCookieName }}
-            - name: ZENML_SERVER_EXTERNAL_COOKIE_NAME
-              value: {{ .Values.zenml.auth.externalCookieName | quote }}
-            {{- end }}
             - name: ZENML_STORE_TYPE
               value: sql
             - name: ZENML_STORE_SSL_VERIFY_SERVER_CERT
               value: {{ .Values.zenml.database.sslVerifyServerCert | default "false" | quote }}
-            {{- if .Values.zenml.secretsStore.enabled }}
-            - name: ZENML_SECRETS_STORE_TYPE
-              value: {{ .Values.zenml.secretsStore.type | quote }}
-            {{- if eq .Values.zenml.secretsStore.type "aws" }}
-            - name: ZENML_SECRETS_STORE_AUTH_METHOD
-              value: {{ .Values.zenml.secretsStore.aws.authMethod | quote }}
-            {{- if .Values.zenml.secretsStore.aws.region_name }}
-            - name: ZENML_SECRETS_STORE_REGION_NAME
-              value: {{ .Values.zenml.secretsStore.aws.region_name | quote }}
-            {{- end }}
-            {{- else if eq .Values.zenml.secretsStore.type "gcp" }}
-            - name: ZENML_SECRETS_STORE_AUTH_METHOD
-              value: {{ .Values.zenml.secretsStore.gcp.authMethod | quote }}
-            {{- if .Values.zenml.secretsStore.gcp.project_id }}
-            - name: ZENML_SECRETS_STORE_PROJECT_ID
-              value: {{ .Values.zenml.secretsStore.gcp.project_id | quote }}
-            {{- end }}
-            {{- if .Values.zenml.secretsStore.gcp.google_application_credentials }}
-            - name: GOOGLE_APPLICATION_CREDENTIALS
-              value: /gcp-credentials/credentials.json
-            {{- end }}
-            {{- else if eq .Values.zenml.secretsStore.type "azure" }}
-            - name: ZENML_SECRETS_STORE_AUTH_METHOD
-              value: {{ .Values.zenml.secretsStore.azure.authMethod | quote }}
-            - name: ZENML_SECRETS_STORE_KEY_VAULT_NAME
-              value: {{ .Values.zenml.secretsStore.azure.key_vault_name | quote }}
-            {{- else if eq .Values.zenml.secretsStore.type "hashicorp" }}
-            - name: ZENML_SECRETS_STORE_VAULT_ADDR
-              value: {{ .Values.zenml.secretsStore.hashicorp.vault_addr | quote }}
-            {{- if .Values.zenml.secretsStore.hashicorp.vault_namespace }}
-            - name: ZENML_SECRETS_STORE_VAULT_NAMESPACE
-              value: {{ .Values.zenml.secretsStore.hashicorp.vault_namespace | quote }}
-            {{- end }}
-            {{- if .Values.zenml.secretsStore.hashicorp.max_versions }}
-            - name: ZENML_SECRETS_STORE_MAX_VERSIONS
-              value: {{ .Values.zenml.secretsStore.hashicorp.max_versions | quote }}
+            {{- range $k, $v := include "zenml.serverEnvVariables" . | fromYaml }}
```

Comment:
The one concern I had when I first saw that you replaced the google credentials mount with env vars is that the libraries won't work without having a file reference with the credentials. I quickly realised that we have now moved to using service connectors instead; neat!